sample/build.gradle

```diff
@@ -50,6 +50,7 @@ android {
             signingConfig signingConfigs.release
 		}
 	}
+    dataBinding.enabled = true
```

Comment:
what is the purpose of dataBinding? We don't have that setting specified in the current Integration guide